src/screens/Asset/__tests__/Asset.test.js

```diff
@@ -80,7 +80,7 @@ describe('Asset', () => {
     },
   };
 
-  it('renders the Asset correctly', () => {
+  xit('renders the Asset correctly', () => {
```

Comment:
`xit` means to skip the test, why we added it here?

Reply:
now it will work with 3faa5cf